sms_939/security/ir.model.access.csv

```diff
@@ -1,5 +1,6 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 sms_notification_full,SMS Notification full access,model_sms_notification,base.group_system,1,1,1,1
 sms_hook_full,SMS hook full access,model_sms_hook,base.group_system,1,1,1,1
+sms_sponsorship_full,SMS sponsorship full access,model_sms_sponsorship,base.group_system,1,1,1,1
```

Comment:
Please also add all accesses to `base.group_user` except for unlink. Employees should be able to log SMS.

Reply:
Done.